content/src/patterns/organizing-work/backlog.md

```diff
@@ -1,10 +1,14 @@
 # Backlog
 
 <summary>
-A **backlog** (to-do-list) is a list of (often prioritized) uncompleted work items (deliverables), or (drivers) that need to be addressed.
+Keep an up-to-date list of things you need to address, so that you can remember them, and use that information to plan and organize your work. 
 </summary>
 
-Consider making backlogs visible, not only to other members of a team, but also to the wider organization.
+{{Definition: A **backlog** (to-do list) is a list of (often prioritized) uncompleted work items (typically a deliverable, requirement, or a driver) that need to be addressed.}}
```

Comment:
replace with macro {{define:backlog}} - that pulls the definition from the glossary